packages/cycle-scripts/template/src/JavaScript/app.test.js

```diff
@@ -1,7 +1,8 @@
-// import assert from 'assert'
+module.exports = replacements => `// import assert from 'assert'
```

Comment:
Take a look at the new test setup in the [one-fits-all flavor](https://github.com/cyclejs-community/create-cycle-app-flavors/tree/master/packages/cycle-scripts-one-fits-all)